### DISCUSSION TITLE
Could you please add a toggle for enabling string duping?

### DISCUSSION THREAD

**Discussion by nostalfinals** [PurpurMC/Purpur#1091] [Feature Requests]
Paper seems fixed string duping via these patches (I think):
- https://github.com/PaperMC/Paper/blob/master/patches/server/0368-Validate-tripwire-hook-placement-before-update.patch
- https://github.com/PaperMC/Paper/blob/master/patches/server/0808-Fix-tripwire-state-inconsistency.patch

Could you please add a toggle to turn off it? String duping is important to players who like vanilla properties.

**Comment by hexadecimal233** [comment]
We need it 
its cool

**Comment by RhythmicSys** [comment]
I feel like this is incredibly unnecessary, string farms are one of the easiest, most abundant farms available in vanilla. Unless you're playing a heavily modified version of the game already, I can't imagine how this would be necessary. At that point, I feel like adding a datapack crafting recipe for wool -> string would make more sense

**Reply by hexadecimal233** [reply]
You can buy emeralds using string and tripwire hook and craft wool
its useful tho

**Reply by RhythmicSys** [reply]
![afkStringFarm](https://user-images.githubusercontent.com/45906780/183996292-a8ee4977-fe84-4678-b081-5417b5d97288.png)
100% vanilla AFK string farm
